src/thor/triplegbuilder.cc

```diff
@@ -1206,6 +1206,11 @@ TripLeg_Edge* AddTripEdge(const AttributesController& controller,
     trip_edge->set_country_crossing(directededge->ctry_crossing());
   }
 
+  // Set forward if requested
+  if (controller(kEdgeForward)) {
+    trip_edge->set_forward(directededge->forward());
```

Comment:
you guys know that this is the direction of the shape not the "traversal" direction